packages/app/src/app/overmind/effects/algoliaSearch.ts

```diff
@@ -0,0 +1,19 @@
+import algoliasearch from 'algoliasearch';
```

Comment:
We could call this file `search`... it is just a principle. Codesandbox does not care about it being `algolia`, it just cares about being able to search for stuff 😄

Reply:
changed :)